README.md

```diff
@@ -6,7 +6,7 @@ Thanks for being interested in contributing to pub! Contributing to a new
 project can be hard: there's a lot of new code and practices to learn. This
 document is intended to get you up and running as quickly as possible. If you're
 looking for documentation on using pub, try
-[dartlang.org](https://www.dartlang.org/tools/pub).
+[pub.dev](https://dart.dev/tools/pub/cmd).
```

Comment:
[pub.dev] -> the [pub tool documentation]

Reply:
Actually, that's redundant. But you get the idea... the link text shouldn't be pub.dev.

Reply:
ack